app/src/main/res/layout/previous_responses_header_item.xml

```diff
@@ -42,7 +42,7 @@
     <FrameLayout
       android:layout_width="0dp"
       android:layout_height="2dp"
-      android:background="@color/color_def_mid_grey"
+      android:background="@color/component_color_shared_previous_responses_header_item_container_text_color"
```

Comment:
This is a background color and not a text-color.

Basically you will need to create two names

`component_color_shared_previous_responses_header_item_text_color `
`component_color_shared_previous_responses_header_item_container_background_color `

And use correctly wherever required.

Reply:
Done.